src/metabase/util.clj

```diff
@@ -430,9 +430,13 @@
   {:style/indent 2}
   ([color-symb x]
    {:pre [(symbol? color-symb)]}
-   ((ns-resolve 'colorize.core color-symb) x))
+   (if (config/config-bool :mb-colorize-logs)
+     ((ns-resolve 'colorize.core color-symb) x)
+     x))
   ([color-symb format-string & args]
-   (format-color color-symb (apply format format-string args))))
+   (if (config/config-bool :mb-colorize-logs)
+     (format-color color-symb (apply format format-string args))
+     (apply format format-string args))))
```

Comment:
Since `mb-colorize-logs` won't change during runtime let's save some CPU cycles and only do the check during initialization:

```clojure
(def ^String ^{:style/indent 2} format-color
  "Like `format`, but uses a function in `colorize.core` to colorize the output.
   COLOR-SYMB should be a quoted symbol like `green`, `red`, `yellow`, `blue`,
   `cyan`, `magenta`, etc. See the entire list of avaliable colors
   [here](https://github.com/ibdknox/colorize/blob/master/src/colorize/core.clj).

     (format-color 'red \"Fatal error: %s\" error-message)"
  (if (config/config-bool :mb-colorize-logs)
    (fn
      ([color-symb x]
       {:pre [(symbol? color-symb)]}
       ((ns-resolve 'colorize.core color-symb) x))
      ([color-symb format-string & args]
       (format-color color-symb (apply format format-string args))))
    (fn
      ([_ x] x)
      ([_ format-string & args] (apply format format-string args)))))
```